android/src/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -187,6 +200,30 @@ private void updateTimeLeft(int seconds)
     UiUtils.setTextAndShow(mTimeHourUnits, hour);
   }
 
+  private void changeTimes(){
+    float s1 = mTimeEstimate.getTextSize(), s2 = mTimeMinuteValue.getTextSize();
+    mTimeEstimate.setTextSize(TypedValue.COMPLEX_UNIT_PX, s2);
+    mTimeHourValue.setTextSize(TypedValue.COMPLEX_UNIT_PX, s1);
+    mTimeMinuteValue.setTextSize(TypedValue.COMPLEX_UNIT_PX, s1);
+    ColorStateList col = mTimeEstimate.getTextColors();
+    mTimeEstimate.setTextColor(mTimeMinuteValue.getTextColors());
+    mTimeMinuteUnits.setTextColor(col);
+    mTimeHourUnits.setTextColor(col);
+    mTimeMinuteValue.setTextColor(col);
+    mTimeHourValue.setTextColor(col);
```

Comment:
I'm not convinced yet that all these calculations are necessary. We have two TextView elements, defined in the layout with custom colors already. What is necessary, is to replace the content in them (and make sure that updates to this content will go into the appropriate top or bottom element).

Reply:
There are actually 4 textview elements in a linear layout(minutevalue minuteunit hour value hour unit) and 5th one is the eta(the one on the bottom by default). It made me uncomfortable too writing all those lines to change colour. 
I'll try to find an alternative.

Reply:
Can layouts be swapped somehow?

Reply:
Or maybe they can be defined (duplicated) in XML, and can be switched by the code?

Reply:
Yes that is a good idea.
We can set the alpha to 0 and switch it on tap?

Reply:
No alpha, please. Views should be made `gone` and `visible`.

Reply:
Okay. Understood